src/main/java/frc/robot/Constants.java

```diff
@@ -41,6 +41,13 @@ public static class Wiring {
         public static final int[] MODULE_CANCODER_IDS    = { 11, 2, 8, 5 };
         public static final int   PIGEON_IMU             = 0;
 
+        //Arm wiring ports + ids
+        public static final int ARM_MOTOR_ID_BASE = 9997;
+        public static final int ARM_MOTOR_ID_ELBOW = 9998;
+        public static final int ARM_FALCON_ID_WRIST = 9999;
```

Comment:
Let's choose the motor CAN IDs (avoiding conflicts with any other IDs our robot uses).